boards.txt

```diff
@@ -14258,3 +14258,104 @@ deneyapkart1A.menu.DebugLevel.verbose=Verbose
 deneyapkart1A.menu.DebugLevel.verbose.build.code_debug=5
 
 ##############################################################
+
+AirM2M_CORE_ESP32C3.name=AirM2M_CORE_ESP32C3
+AirM2M_CORE_ESP32C3.vid.0=0x303a
+AirM2M_CORE_ESP32C3.pid.0=0x1001
+
+AirM2M_CORE_ESP32C3.upload.tool=esptool_py
+AirM2M_CORE_ESP32C3.upload.maximum_size=1310720
+AirM2M_CORE_ESP32C3.upload.maximum_data_size=327680
+AirM2M_CORE_ESP32C3.upload.flags=
+AirM2M_CORE_ESP32C3.upload.extra_flags=
+AirM2M_CORE_ESP32C3.upload.use_1200bps_touch=false
+AirM2M_CORE_ESP32C3.upload.wait_for_upload_port=false
+
+AirM2M_CORE_ESP32C3.serial.disableDTR=false
+AirM2M_CORE_ESP32C3.serial.disableRTS=false
+
+AirM2M_CORE_ESP32C3.build.tarch=riscv32
+AirM2M_CORE_ESP32C3.build.target=esp
+AirM2M_CORE_ESP32C3.build.mcu=esp32c3
+AirM2M_CORE_ESP32C3.build.core=esp32
+AirM2M_CORE_ESP32C3.build.variant=esp32c3
```

Comment:
almost missed this. If you want to use the variant header that you added here, you need to change the variant to the folder name. In your case that is `AirM2M_CORE_ESP32C3`

Reply:
I have fixed it, please check